tests/generators/math.xml

```diff
@@ -1881,4 +1886,36 @@
       </block>
     </statement>
   </block>
+  <block type="procedures_defnoreturn" y="5938" x="13">
+    <field name="NAME">test atan2</field>
+    <comment w="160" h="80" pinned="false">Describe this function...</comment>
+    <statement name="STACK">
+      <block type="unittest_assertequals">
+        <value name="MESSAGE">
+          <block type="text">
+            <field name="TEXT">atan2</field>
+          </block>
+        </value>
+        <value name="ACTUAL">
+          <block type="math_atan2">
+            <value name="X">
+              <block type="math_number">
+                <field name="NUM">-1</field>
```

Comment:
Choose a different values for X and Y so the test can ensure the order of arguments is correct in each generated language.

Reply:
Also, consider adding a test with a result in the range of 90 < x < 180, so we are sure the range of the result is -180 to 180 in all languages.